feature-toggle/src/app/demo/demo.component.html

```diff
@@ -0,0 +1,12 @@
+<p>demo works!</p>
+<button mat-raised-button (click)="openDialog()"> 
+    Pick one 
+</button> 
+<br> 
+ 
+
+@if (business) {
+<span style="background-color: brown;"> 
```

Comment:
use div if we are expecting it to be in next line

Reply:
deleted component demo